pyomo/contrib/trustregion/PyomoInterface.py

```diff
@@ -94,6 +123,7 @@ class PyomoInterface:
 
     '''
 
+    TempfileManager.tempdir = "."
```

Comment:
We do NOT want to be mucking with the tempfile directory....  I think I removed this line from master -- and you do not want to re-add it. (it breaks other tests and does evil things with polluting a user's current working directory.